src/main/java/com/codessquad/qna/controller/QuestionController.java

```diff
@@ -36,4 +38,14 @@ public String showQuestionDetail(@PathVariable Long index, Model model) {
         return "/qna/show";
     }
 
+    @GetMapping("/qna/form")
+    public String qnsForm(HttpSession session) {
+        if(HttpSessionUtils.isLoginUser(session)) {
+
+            return "/qna/form";
```

Comment:
```suggestion
        if (HttpSessionUtils.isLoginUser(session)) {
            return "/qna/form";
```

빈 라인도 코드입니다. 신중하게 작성해주세요.
코드 커밋 전 정렬을 수행해주세요.

Reply:
완료했습니다